app/test/testmerginapi.cpp

```diff
@@ -68,27 +68,30 @@ void TestMerginApi::testDownloadProject()
   QSignalSpy spy( mApi, SIGNAL( syncProjectFinished( QString, QString, bool ) ) );
   QString projectName = "mobile_demo_mod"; // TODO depends on mergin test server, unless a project is created beforehand
   QString projectNamespace = mUsername; // TODO depends on mergin test server, unless a project is created beforehand
-  mApi->downloadProject( mUsername + "/" + projectName );
+  mApi->downloadProject( projectNamespace, projectName );
 
   QVERIFY( spy.wait( 5000 ) );
   QCOMPARE( spy.count(), 1 );
 
   ProjectList projects = mMerginProjectModel->projects();
   Q_ASSERT( !mMerginProjectModel->projects().isEmpty() );
-  projectName = mMerginProjectModel->projects().at( 0 )->name;
+
+  bool downloadSuccessful = mProjectModel->containsProject( projectName );
+  Q_ASSERT( downloadSuccessful );
```

Comment:
tests should use QVERIFY instead of Q_ASSERT - the latter aborts the whole thing while the former just aborts the one test method and gives a nice error message